base/range.jl

```diff
@@ -182,33 +183,33 @@ end
 
 function linspace{T<:AbstractFloat}(start::T, stop::T, len::T)
```

Comment:
Isn't the problem really that `len` shouldn't have units for this definition to make sense?